src/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -263,7 +262,7 @@
                  max-frame-size threads]} command-processing
         {:keys [certificate-whitelist
                 disable-update-checking]} puppetdb
-        url-prefix (get-route service)
+        url-prefix "/pdb";;(get-route service)
```

Comment:
This is one of the causes of the WIP, I'm spending time now figuring out how to remove this.